config/features.yml

```diff
@@ -1871,4 +1875,3 @@ features:
   is_DGIB_endpoint:
     actor_type: user
     description: used to call data from DGIB endpoints for MGIB VYE application
-
```

Comment:
Please add new line to end of file

Reply:
Should there be two new lines at the end of the file? It looks like on master there are two new lines and my auto linter removed one. I can add it back but I think one is what we would want?

Reply:
oh yeah, I see what you mean. this is fine 👍 thanks 
<img width="228" alt="image" src="https://github.com/user-attachments/assets/9adf2171-e35b-4180-8189-a34bb68db194">